ToDoList/ToDoList/Modules/TaskDisplay/Core/Presenter/TaskDisplayPresenter.swift

```diff
@@ -25,22 +25,32 @@ final class TaskDisplayPresenter: TaskDisplayPresenterInterface {
     
     func getTasksBySection(with section: TaskStatus) -> [TaskItem]? {
         guard let tasks = storage.getTasks() else { return nil }
-        return tasks.filter { task in
-            task.status == section
-        }
+        let sectionedTasks = tasks.filter({$0.status == section})
```

Comment:
```suggestion
        let sectionedTasks = tasks.filter { $0.status == section }
```
Just a preference, resolve this comment if you like it your way 😄

Reply:
yeah, I forgot to activate lint, will fix it right now